### PR TITLE
Bug 1768769: Update all pipeline default create sample

### PR DESCRIPTION
**Pull request by vikram-raj**
Bug - https://jira.coreos.com/browse/ODC-2197

### CODE REVIEW

**Comment by spadgett** [comment]
This is close to fixing Bug 1768769, but would need PipelineResource and ClusterTask examples.

**Comment by spadgett** [comment]
cc @andrewballantyne

**Comment by vikram-raj** [comment]
@spadgett @andrewballantyne  I am adding the default create a sample for ```PipelineResource``` and ```ClusterTask```. I missed these two resources as it is not visible in the dev prespective.

**Comment by andrewballantyne** [comment]
For posterity, I posted on the ticket how to locate these resources. You can use the Search to find any page that is not in the navbar in the dev perspective.

Thanks for the tag @spadgett

**Comment by vikram-raj** [comment]
@andrewballantyne @spadgett Added default create sample for ```PipelineResource``` and ```ClusterTask``` as well.

File: frontend/packages/dev-console/src/templates/index.ts [hunk 1]
```diff
@@ -1 +1,5 @@
 export * from './pipeline';
+export * from './task';
+export * from './taskrun';
+export * from './pipelineresource';
+export * from './clustertask';
```
Comment (andrewballantyne) [inline frontend/packages/dev-console/src/templates/index.ts]: Snake case please - files are either `PascalCase` if they are classes/components or `snake-case` if they are utilities, folders or general help files.

File: frontend/packages/dev-console/src/templates/index.ts [hunk 2]
```diff
@@ -1 +1,5 @@
 export * from './pipeline';
```
Comment (andrewballantyne) [inline frontend/packages/dev-console/src/templates/index.ts]: I wonder if we should just rename this to `pipelines` and put everything in here. Each with a single (non-default) export seems a bit weird.

**Comment by vikram-raj** [comment]
@andrewballantyne @siamaksade I am was trying to run the pipeline by just creating all the resources from the sample without making any changes. But I am not able to do it as an image is not define in ```pipelineResources``` sample. Posted this on gist and slack. https://gist.github.com/siamaksade/21a457868b9662ca6ae58d564d81ede7#gistcomment-3081154

**Comment by andrewballantyne** [comment]
/lgtm

The decision was made not to concern ourselves with this at this time. @vikram-raj thanks for looking into this.

**Comment by andrewballantyne** [comment]
/lgtm cancel

Whoops, still waiting on the file changes.

File: frontend/packages/dev-console/src/templates/index.ts (hunk 2 above)
Comment (vikram-raj) [reply frontend/packages/dev-console/src/templates/index.ts]: Agree, we can put everything in one file.

Comment (vikram-raj) [reply frontend/packages/dev-console/src/templates/index.ts]: Done.

**Comment by spadgett** [comment]
> error An unexpected error occurred: "https://registry.yarnpkg.com/@fortawesome/fontawesome-free/-/fontawesome-free-5.9.0.tgz: ESOCKETTIMEDOUT".

/retest

**Pull request merged**